examples/undocumented/libshogun/classifier_mklmulticlass.cpp

```diff
@@ -1,13 +1,11 @@
 /*
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or
- * (at your option) any later version.
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
  *
- * Written (W) 2009 Alexander Binder
- * Copyright (C) 2009 Fraunhofer Institute FIRST and Max-Planck-Society
+ * Authors: Soeren Sonnenburg, D. Lehmann, Evgeniy Andreev, Thoralf Klein,
```

Comment:
D. Lehmann should not be in any list due to being D. Lehmann

Reply:
yep just remove